tools/migrate_to_bzlmod.py

```diff
@@ -112,6 +112,26 @@ def scratch_file(file_path, lines=None, mode="w"):
     return abspath
 
 
+def append_to_file(filename, content):
+    """
+    Creates a file with the given filename and content.
+
+    Args:
+        filename (str): The name of the file to create.
+        content (str): The content to write to the file.
+    """
+    try:
+        with open(filename, "a") as f:
+            f.write(content)
+    except OSError as e:
+        error(f"Error creating file '{filename}': {e}")
+
+
+def append_migration_info(content):
+    """Adds content to the "migration_info" file in order to help users with details about the migration."""
+    append_to_file("migration_info.md", content + "\n")
```

Comment:
The `migration_info.md` file is re-created during re-run, can we preserve its content?

Reply:
I agree. Done, thanks for the catch!